### PR TITLE
[bug 1134830] Update README

### PR DESCRIPTION
**Pull request by ckprice**
- Add Map Data source to README
- Add notes from original agency README (https://github.com/chrisjames-work/lorax)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,68 @@ to set up a local development copy of Lorax, check out the
...
+
+The base/ folder contains all data that is consistent across languages. None of the data in this folder will be changed as a result of localization. 
+
+##### main.json
+Contains all issues, grouped by topic. Data for styling, type of infographic, tags, etc.
+
+##### countries.topo.json
+Contains the SVG data for drawing the world map.
+
+### i18N
```
Comment (schalkneethling) [inline README.md]: Do we still need i18n seeing all the strings live in bedrock or, are there some additional strings that needs to be translated in this repo?

File: README.md [hunk 2]
```diff
@@ -13,6 +13,68 @@ to set up a local development copy of Lorax, check out the
...
+
+i18n is the localization folder. In it are subfolders which represent each language. Each language folder contains several files:
+
+##### main.json
+This file has four main sections: **topics**, **modals**, and **misc**. <br>
```
Comment (schalkneethling) [inline README.md]: three sections?

**Comment by schalkneethling** [comment]
just nits

**Comment by ckprice** [comment]
@schalkneethling updated, thanks!

**Comment by schalkneethling** [comment]
lgtm r+

**Pull request merged**